localstack/http/asgi.py

```diff
@@ -110,10 +110,24 @@ def _next_sync():
         yield val
 
 
-class HTTPRequestEventStreamAdapter:
+def create_wsgi_input(
+    receive: "ASGIReceiveCallable", event_loop: t.Optional[AbstractEventLoop] = None
+) -> t.IO[bytes]:
     """
-    An adapter to expose an ASGIReceiveCallable coroutine that returns HTTPRequestEvent
-    instances, as a PEP 3333 InputStream for consumption in synchronous WSGI/Werkzeug code.
+    Factory for exposing an ASGIReceiveCallable as an IO stream.
+    :param receive:
+    :param event_loop:
+    :return:
```

Comment:
nit: This looks unfinished, so either we need some information here or remove the section all together :)